src/main/scala/vectorpipe/ProcessOSM.scala

```diff
@@ -61,7 +64,21 @@ object ProcessOSM {
       StructField("geom", GeometryUDT) ::
       Nil)
 
-  val VersionedElementEncoder: Encoder[Row] = RowEncoder(VersionedElementSchema)
+  lazy val VersionedElementEncoder: Encoder[Row] = RowEncoder(VersionedElementSchema)
+
+  /* support UDF for constructGeometries */
+  private case class StrMember(`type`: String, ref: Long, role: String)
+  private val convertMembers = org.apache.spark.sql.functions.udf { member: Seq[Row] =>
```

Comment:
This already exists, albeit in a slightly different form: https://github.com/geotrellis/vectorpipe/pull/60/files#diff-28c19e0cb428278057965fad1a00e689R183